language-ids.sublime-settings

```diff
@@ -21,103 +21,38 @@
-    "source.jsx.unittest": "javascriptreact", // https://github.com/SublimeText/AFileIcon
     "source.Kotlin": "kotlin", // https://github.com/vkostyukov/kotlin-sublime-package
     "source.modern-fortran": "fortran", // https://packagecontrol.io/packages/Fortran
     "source.objc": "objective-c",
     "source.objc++": "objective-cpp",
     "source.shader": "shaderlab", // https://github.com/waqiju/unity_shader_st3
-    "source.shell.bash": "shellscript",
-    "source.shell.docker": "shellscript", // https://github.com/SublimeText/AFileIcon
-    "source.shell.eslint": "shellscript", // https://github.com/SublimeText/AFileIcon
-    "source.shell.npm": "shellscript", // https://github.com/SublimeText/AFileIcon
-    "source.shell.ruby": "shellscript", // https://github.com/SublimeText/AFileIcon
-    "source.shell.stylelint": "shellscript", // https://github.com/SublimeText/AFileIcon
+    "source.shell": "shellscript",
     "source.ts": "typescript",
     "source.ts.react": "typescriptreact", // https://github.com/Thom1729/Sublime-JS-Custom
-    "source.ts.unittest": "typescript", // https://github.com/SublimeText/AFileIcon
     "source.tsx": "typescriptreact",
-    "source.tsx.unittest": "typescriptreact", // https://github.com/SublimeText/AFileIcon
     "source.unity.unity_shader": "shaderlab", // https://github.com/petereichinger/Unity3D-Shader
-    "source.viml.vimrc": "viml", // https://github.com/SublimeText/AFileIcon
+    "source.viml": "viml",
     "source.yaml-tmlanguage": "yaml", // https://github.com/SublimeText/PackageDev
-    "source.yaml.circleci": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.docker": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.eslint": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.lock": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.procfile": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.stylelint": "yaml", // https://github.com/SublimeText/AFileIcon
-    "source.yaml.sublime.syntax": "yaml", // https://github.com/SublimeText/PackageDev
-    "source.yaml.yarn": "yaml", // https://github.com/SublimeText/AFileIcon
+    "source.yaml": "yaml",
     "text.advanced_csv": "csv", // https://github.com/SublimeText/AFileIcon
     "text.django": "html", // https://github.com/willstott101/django-sublime-syntax
-    "text.html.basic": "html",
-    "text.html.elixir": "html",  // https://github.com/elixir-editors/elixir-tmbundle
-    "text.html.markdown.academicmarkdown": "markdown", // https://github.com/mangecoeur/AcademicMarkdown
-    "text.html.markdown.license": "markdown", // https://github.com/SublimeText/AFileIcon
+    "text.html": "html",
+    "text.html.markdown": "markdown",
     "text.html.markdown.rmarkdown": "r", // https://github.com/REditorSupport/sublime-ide-r
-    "text.html.ngx": "html", // https://github.com/princemaple/ngx-html-syntax
+    "text.html.vue": "vue",
```

Comment:
Looking through the language IDs from the specs, two other scopes where it breaks are `text.html.handlebars` -> `handlebars` and `text.xml.xsl` -> `xsl`.

Could you also add `"source.ruby": "ruby"` here, then I could remove the [settings file](https://github.com/jwortmann/brackets-theme/blob/e72123f57a478c2cde9c382b6eb0bbe2ed338cf2/settings/language-ids.sublime-settings) from my theme.

Reply:
Off topic, 
I didn't know there was a brackets theme for ST <3

Brackets was my first editor I enjoyed using. 
It had a really unique look, and the theme really looks like Brackets(only the tabs should be hidden ;) )